libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -450,46 +465,65 @@ function BallotMeasureContest({
               tableBorderColor={Colors.DARKER_GRAY}
               tableHeaderBackgroundColor={Colors.LIGHT_GRAY}
             >
-              {electionStrings.contestDescription(contest)}
```

Comment:
I couldn't think of any easy way to make this work with translations, and since NH ballots are mono-lingual for now, I punted. But I feel a bit worried about this in the future

Reply:
Comment added: fee440576